playground/css/__tests__/tests.ts

```diff
@@ -539,4 +540,13 @@ export const tests = (isLightningCSS: boolean) => {
     expect(await getColor('.treeshake-scoped-order')).toBe('red')
     expect(await getBgColor('.treeshake-scoped-order')).toBe('blue')
   })
+
+  test.runIf(isBuild)(
+    'empty CSS files should generate .css assets, not .js assets',
+    () => {
+      const manifest = readManifest()
+      expect(manifest['empty.css']).toBeDefined()
+      expect(manifest['empty.css'].file).toMatch(/\.css$/)
+    },
+  )
```

Comment:
Do we need to use the manifest to validate the file extension? Could we check the output assets directly like this?

https://github.com/vitejs/vite/blob/e49f50599d852eec644e79b074b4648e2dff1e5d/playground/css/__tests__/tests.ts#L519-L524